### PR TITLE
Refactor features tables

### PR DESCRIPTION
**Pull request by palemieux**
Add support for negative values (#370, #394)
Clarify support for #textOrientation-sideways-LR (#373)
Added #extent-length (#382)
Move tts:position constraints to Text Profile (#392)
Fix localBiblio bug
Refactor feature lists per TTML2 CR2 (#390, #393)
Replace `tts:fontShear` with `tts:shear` and `tts:lineShear` (#401)

### CODE REVIEW

File: imsc1/spec/ttml-ww-profiles.html [hunk 1]
```diff
@@ -629,37 +681,69 @@ <h2>Supported Features and Extensions</h2>
...
+          <td colspan="2">See individual disposition of <code>#extent-root</code> and <code>#extent-auto-version-2</code>.</td>
+        </tr>
+
         <tr>
           <td><code>#extent-version-2</code></td>
 
-          <td colspan="2">See individual disposition of <code>#extent</code>, <code>#extent-block</code>,
-          <code>#extent-image</code>, and <code>#extent-measure</code>.<br>
-          <small>Section <a href="#extent-version-2"></a> specifies additional constraints.</small></td>
+          <td colspan="2">See individual disposition of <code>#extent</code>m <code>#extent-image</code>, and
```
Comment (nigelmegitt) [inline imsc1/spec/ttml-ww-profiles.html]: s/m/,

**Comment by palemieux** [comment]
Built at https://rawgit.com/w3c/imsc/issues-refactor-features-table/imsc1/spec/ttml-ww-profiles.html

**Comment by spoeschel** [comment]
Hmm, I'm not sure if I have overseen something, but shouldn't `#extent-auto` be permitted as in IMSC 1.0? And thus also `#extent-region` and `#extent-root`?

**Comment by spoeschel** [comment]
`#length` in the table in §6 refers to several dependent features. However in the current TTML2 draft that feature doesn't list any dependent features.

**Comment by spoeschel** [comment]
@nigelmegitt Hmm, I see. It looks a bit odd to me to say "See individual disposition of (...)" here, as the listed features here all refer to TTML2 features of that name. While the actual `#length` definition as part of TTML1 refers to TTML1 features of the same name in the end. All the affected, dependent TTML2 features refer to their TTML1 counterpart, however I'm not really in favour of that jumping back and forth between TTML2 and TTML1. But it's fine to keep it this way.

Nevertheless `#length` has to be changed to "permitted" for the Text profile, as all dependent features are permitted for that profile.

**Comment by palemieux** [comment]
@spoeschel `#extent-auto` is effectively prohibited in IMSC1 since `tts:extent` on `tt` is always in `px`, and `tts:extent` on `region` is constrained to `px` or `%`.

**Comment by palemieux** [comment]
> Nevertheless #length has to be changed to "permitted" for the Text profile, as all dependent features are permitted for that profile.

@spoeschel I have made the change.

**Comment by spoeschel** [comment]
@palemieux:
Ah OK, I see. Thanks for fixing `#length`.

**Comment by nigelmegitt** [comment]
I see that this pull request allows `position` to have `em` units but does not allow `origin` to - I think this is a mistake and have raised #404 (which as an aside is obviously the best issue number).

**Comment by palemieux** [comment]
@nigelmegitt See https://github.com/w3c/imsc/pull/397

**Comment by nigelmegitt** [comment]
Thanks @palemieux for pointing me at #397 that is a duplicate of #404 - I've closed the latter.

**Comment by nigelmegitt** [comment]
Reviewing this pull request against #393, I see that there are mentioned features in a "See disposition for XXX" style that have no entry in the features table. I think that's confusing and would rather simply say something like "partially supported via the dependent features [supported features list]" and omit the unsupported dependent features altogether.

**Comment by palemieux** [comment]
@nigelmegitt Do you think we can merge this and handle https://github.com/w3c/imsc/pull/395#issuecomment-400760541 as a separate issue?

**Pull request merged**